OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix for reports component Phase 7

# Description
Lint fix for Reports component, this is phase 7.

## Files changed
All js/jsx file in /src/components/Reports/TasksDetail

eslint test:
<img width="1566" alt="CleanShot 2023-09-30 at 18 38 58@2x" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/f62c5912-01aa-41bd-8f9c-8ac8d2268eeb">




# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint src/components/Reports/TasksDetail/TasksDetail.jsx`
- `npx eslint fix src/components/Reports/TasksDetail/index.js`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!
